product/import_products_big_query_table.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import time
+
+from google.api_core.client_options import ClientOptions
+from google.cloud.retail_v2 import BigQuerySource, ProductInputConfig, ProductServiceClient, \
+    ImportProductsRequest
+
+# TODO Define the project number here:
+project_number = ""
+
+endpoint = "retail.googleapis.com"
+default_catalog = "projects/{0}/locations/global/catalogs/default_catalog/branches/1".format(project_number)
+project_id = "crs-interactive-tutorials"
+dataset_id = "products"
+table_id = "products_for_import"
+
+
+# [START product_client]
+def get_product_service_client():
+    client_options = ClientOptions(endpoint)
+    return ProductServiceClient(client_options=client_options)
+    # [END product_client]
+
+
+# [START get_import_products_big_query_request]
+def get_import_products_big_query_request(reconciliation_mode):
```

Comment:
I don't see any reason to have reconciliation mode as parameter if it's hardcoded in next method.